yugabyte_cloudformation.yaml

```diff
@@ -0,0 +1,490 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: "Simple deployment of YugaByte DB cluster in multi-AZ (default availability zone is 3 and replication factor is also 3)"
+
+Parameters: 
+  DBVersion:
+    Description: YugaByte DB version Default version is 1.2.8.0
+    Type: String
+    Default: "1.2.8.0"
+  KeyName:
```

Comment:
we need to parameterize the replication factor and number of nodes as well

Reply:
We can parameterize the replication factor but we can't parameterize the number of nodes because in CloudFormation template we can not specify the number of EC2 instances dynamically.

Reply:
This is fine for now. We can write docs on how to change the num nodes. Also, about replication factor, lets open a github issue and move on. Trying to get the default flow up and running asap.